ci/test-ault-fujitsu.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
```

Comment:
set -euxo pipefail

Reply:
I don't understand the comment?

Reply:
Cf. https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
ok, good to know. We don't need the `-x`, because gitlab prints those out nicely for us.
`set -euo pipefail` looks a good option.